scripts/gen_merkle_dist.js

```diff
@@ -1,56 +1,119 @@
 // Script that generates a new Merkle Distribution and outputs the data to JSON files
 
 const fs = require("fs")
+const shell = require("shelljs")
+const dotenv = require("dotenv").config()
 const Subgraph = require("../src/stakingrewards/subgraph.js")
 const Rewards = require("../src/stakingrewards/rewards.js")
 const MerkleDist = require("../src/stakingrewards/merkle_dist.js")
 
+const bonusWeight = 0.0
+const ongoingWeight = 0.875
+const tbtcv2Weight = 0.125
```

Comment:
I'm not 100% sure, but if the rewards cover most of October, then shouldn't it be more like 0.75 and 0.25 according to this https://blog.threshold.network/tbtc-v2-hits-its-first-launch-milestone/?